downstream/assemblies/platform/assembly-controller-teams.adoc

```diff
@@ -1,24 +1,31 @@
 ifdef::context[:parent-context: {context}]
 
-[id="assembly-controller-teams"]
+:_mod-docs-content-type: ASSEMBLY
+
+[id="assembly-controller-teams_{context}"]
+
+= Teams
 
 :context: controller-teams
-= Managing teams
 
-A *Team* is a subdivision of an organization with associated users, projects, credentials, and permissions.
-Teams offer a means to implement role-based access control schemes and delegate responsibilities across organizations.
-For example, you can grant permissions to a whole team rather than to each user on the team.
+A team is a subdivision of an organization with associated users, and resources. Teams provide a means to implement role-based access control schemes and delegate responsibilities across organizations. For instance, you can grant permissions to a Team rather than each user on the team.
+You can create as many teams as needed for your organization. Teams can only be assigned to one organization while an organization can be made up of multiple teams. Each team can be assigned roles, the same way roles are assigned for users. Teams can also scalably assign ownership for credentials, preventing multiple interface click-throughs to assign the same credentials to the same user.
```

Comment:
Rewrite of the last sentence in this line for clarity:
With teams you can assign credential ownership at scale, preventing the need to click through multiple interfaces to assign the same credentials to each individual user.
(if that's not what you were trying to say please disregard)

Reply:
Fixed.